pkg/cluster/streams.go

```diff
@@ -0,0 +1,261 @@
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"strings"
+
+	acidv1 "github.com/zalando/postgres-operator/pkg/apis/acid.zalan.do/v1"
+	zalandov1alpha1 "github.com/zalando/postgres-operator/pkg/apis/zalando.org/v1alpha1"
+	"github.com/zalando/postgres-operator/pkg/util"
+	"github.com/zalando/postgres-operator/pkg/util/config"
+	"github.com/zalando/postgres-operator/pkg/util/constants"
+	"github.com/zalando/postgres-operator/pkg/util/k8sutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var outboxTableNameTemplate config.StringTemplate = "{table}_{eventtype}_outbox"
+
+func (c *Cluster) createStreams() error {
+	c.setProcessName("creating streams")
+
+	fes := c.generateFabricEventStream()
+	_, err := c.KubeClient.FabricEventStreams(c.Namespace).Create(context.TODO(), fes, metav1.CreateOptions{})
```

Comment:
Is the Zalando-internal custom operator assumed to always listen to all namespaces ? in other words, does `FabricEventStreams(c.Namespace)` always make sense ?

Reply:
the FES CRD is created in the cluster's namespace. You think somebody might wish to create it in a separate namespace?